src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -405,17 +405,67 @@ EscrowCreate::preclaim(PreclaimContext const& ctx)
     return tesSUCCESS;
 }
 
+template <ValidIssueType T>
+static TER
+escrowLockApplyHelper(
+    ApplyView& view,
+    AccountID const& issuer,
+    AccountID const& account,
```

Comment:
not `SenderID` like in `escrowUnlock`?

Reply:
I renamed them all to issuer, sender, receiver. No `u` or `ID`. But you can see in the `rippleUnlockEscrowMPT` function I changed it to grantor and grantee and I think these better reflect the accounts because sender is not the sender of the transaction but the account that created the escrow. So I think they should be changed in this helper function too but I really don't care about the names of variables. So if you want me to change them in the `rippleLockEscrowMPT` and `rippleUnlockEscrowMPT` functions I don't mind.

Reply:
Yes, please, it is easier to understand connections among thoise functions when they use the same names for the accounts

Reply:
Ok they are all issuer, sender and receiver now.